README.rst

```diff
@@ -2,3 +2,169 @@
  REANA example - CMS reconstruction
 ====================================
 
+.. image:: https://img.shields.io/travis/reanahub/reana-demo-cms-reco.svg
+   :target: https://travis-ci.org/reanahub/reana-demo-cms-reco
+
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+   :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+
+.. image:: https://img.shields.io/github/license/reanahub/reana-demo-cms-reco.svg
+   :target: https://raw.githubusercontent.com/reanahub/reana-demo-alice-cms-reco/master/LICENSE
+
+
+About
+======
+This REANA reproducible analysis example demonstrates the reconstruction
+procedure of the CMS collaboration from raw data to `Analysis Object Data (AOD) <https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookDataFormats#AoD>`_.
+
+The workflow consists of the steps need for the samples reconstruction, as taken
+from the `CMS legacy validation repo <https://github.com/cms-legacydata-validation/RAWToAODValidation/tree/master>`_.
+
+Analysis structure
+==================
+
+Making a research data analysis reproducible basically means to provide
+"runnable recipes" addressing (1) where is the input data, (2) what software was
+used to analyse the data, (3) which computing environments were used to run the
+software and (4) which computational workflow steps were taken to run the
+analysis. This will permit to instantiate the analysis on the computational
+cloud and run the analysis to obtain (5) output results.
+
+
+1 & 2. Input data and Analysis code
+------------------------------------
+
+Any raw input data from the `CERN open data platform <http://opendata.cern.ch/search?page=1&size=20&experiment=CMS&type=Dataset&subtype=Collision&subtype=Derived&subtype=Simulated&file_type=raw>`_
+should be valid for reconstruction.
+
+The reconstruction step can be repeated with a configuration file that depends
+on the analyzed data, e.g. `this example <http://opendata.cern.ch/record/43>`_.
```

Comment:
For the first commit, pick some dataset and use that example.

This will be anyway all amended later when the workflow factory is created.